src/config.ts

```diff
@@ -210,3 +211,6 @@ if (CSP_NONCE) {
     delete currentScriptDataset.nonce;
   }
 }
+
+export const ALLOW_INLINE_CSS = inlineCss !== 'off';
+export const EXTERNALIZE_INLINE_CSS = inlineCss === 'data:';
```

Comment:
could you clarify why we check against `data:` here? it doesnt look like we pass the `inlineCss` value anywhere, so it seems to me to just be an odd enum value

Reply:
these are options for this parameter. data: is the name of the externalization option atm to afford for a potential future blob: option etc. to best fit varying CSPs

Reply:
airgap.js forwards all of its loadOptions to modules that it loads, so the UI module automatically gets this value